pkg/datapath/types/node.go

```diff
@@ -181,6 +181,10 @@ type LocalNodeConfiguration struct {
 	// EncryptNode enables encrypting NodeIP traffic
 	EncryptNode bool
 
+	// EnablePolicyAccounting
```

Comment:
This could read more naturally:

```suggestion
	// EnablePolicyAccounting enables maintaining packet and byte counters for every
	// policy entry.
```

Reply:
ok, i will fix it